addons/web/static/src/scss/bootstrap_overridden_frontend.scss

```diff
@@ -93,6 +93,10 @@ $form-range-thumb-bg: $body-bg !default;
 $form-range-thumb-active-bg: $primary !default;
 $form-range-thumb-disabled-bg: $input-disabled-bg !default;
 
+$btn-hover-bg-shade-amount: 15% !default;
```

Comment:
Almost there 💪

By convention a `bootstrap_overridden*` file "_is a copy of the bootstrap variables.scss file where all the left-untouched variables definition have been removed_",  meaning that the **order** of the variable we define here should match BS default (see https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss).

You should move `$btn-hover-bg-shade-amount`definition around line 68, in a new "section" called `// Buttons`, just before `// Forms`.